.ruff.toml

```diff
@@ -422,3 +421,70 @@ inline-quotes = "single"
 forced-separate = [
     "tests",
 ]
+
+[format]
+quote-style = "single"
```

Comment:
I'm in favour of using the defaults

Reply:
Usually I also would be, but:

1. this is not compatible with the `flake8-quotes` configuration already set above
2. the majority of the existing code base is already formatted as single quotes (enforced by flake8) and so this would enforce an unnecessarily larger diff to change current files

Reply:
I'm still personally in favour of using the defaults.
I feel that if you're going to standardise the formatting you might as well go all the way

Reply:
This can be discussed at a later date, but it will not be changed here, since it specifically goes against this PRs goal of not enforcing changes on any current files or linting